cmds/task/create.go

```diff
@@ -0,0 +1,138 @@
+package task
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/taskcluster/slugid-go/slugid"
+	"github.com/taskcluster/taskcluster-cli/config"
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+)
+
+var (
+	createCmd = &cobra.Command{
+		Use:     "create <image> <command>",
+		Short:   "Creates and schedules a task, using the specified taskId if given.",
+		PreRunE: checkCreateFlags,
+		RunE:    runCreateTask,
+	}
+
+	now = time.Now().UTC()
+
+	createPayload = &queue.TaskDefinitionRequest{
+		SchedulerID: "taskcluster-cli",
+
+		Created:  tcclient.Time(now),
+		Deadline: tcclient.Time(now.Add(24 * time.Hour)),
+		Expires:  tcclient.Time(now.Add(24*time.Hour).AddDate(1, 0, 0)),
```

Comment:
SchedulerId is the ID of the scheduler that defined the task, correct? I considered that as set because these tasks will always be defined by `taskcluster-cli`.

I'll see what I can do for the deadline and expires parameters